scripts/deploy.sh

```
@@ -0,0 +1,159 @@
+
+sync_env "$ENV_FILE"
+sync_env "$ENV_LOCAL_FILE"
+
+echo "✅ Environment files updated: $ENV_FILE, $ENV_LOCAL_FILE"
+
+# Verification
+echo ""
+echo "🔍 Running basic verification..."
+# Call a getter to ensure it works
+PROMPTS_COUNT=$(stellar contract invoke \
+    --id $CONTRACT_ID \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)